docs/release-notes/NuGet-5.9.md

```diff
@@ -207,6 +207,15 @@ Thank you to all the contributors who helped make this NuGet release awesome!
 [xen2](https://github.com/xen2) | [2861](https://github.com/NuGet/NuGet.Client/pull/2861) | Make NuGet.PackageManagement available as a .NET Standard package - [#6150](https://github.com/NuGet/Home/issues/6150)
 [Anipik](https://github.com/Anipik) | [3810](https://github.com/NuGet/NuGet.Client/pull/3810) | Introduce a new msbuild property to exclude build output for specific tfms during pack task - [#10396](https://github.com/NuGet/Home/issues/10396)
 
+## Summary: What's New in 5.9.1
+
+* "dotnet nuget remove source nuget.org" doesn't work the first time - [#10745](https://github.com/NuGet/Home/issues/10745)
+* Make default validation disabled on Linux, but enabled by default on Windows - [#10713](https://github.com/NuGet/Home/issues/10713)
+
+**[List of all issues fixed in this release - 5.9.1](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=6075f42efd068017639b4036)**
```

Comment:
Just pointing this out that this will not be publicly available. I know we added it in 5.9 & it's fine for 5.9.1 but just commenting for history later on!

Reply:
Yeah, this has been some feedback from customers too. 
Hopefully with zenhub sprints, we'll be able to start using the GH milestone field instead.